packages/aws-cdk/README.md

```diff
@@ -559,6 +560,87 @@ This feature is currently in preview. Be aware of the following limitations:
   in the right order. The CLI will not help you import dependent resources in the right
   order, the CloudFormation deployment will fail with unresolved references.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
```

Comment:
It's fine to open with this, but immediately below should be a paragraph explaining what this command does at a high level, including an example of simplest, minimal usage, eg only using required flags / options. It should explain where the output goes and what the output is. I should not have to guess if it's a full CDK directory or if it's a single stack file.